woocommerce/woocommerce-ios
# [Mobile Payments] Card Reader Settings 2-9: Hook up card reader connect

Closes #4055 

Note: This PR is against feature/stripe-terminal-sdk-integration not develop

Note: This is the last PR in the Card Reader Settings v2 effort before we remove the v1 code ( 2-12 #4058 )

Changes:
- Adds new images for connecting and finding readers, including dark mode images
- Adds new modals for a found reader and connecting to a reader
- Allows the user to connect to a found reader, or continue searching

To test:
- Settings > Manage Card Readers v2 > Connect Card Reader
- Power on your reader
- The modal should change from a "Scanning for reader" modal to a "Do you want to connect to..." modal
- Tap Connect to Reader
- The view should change to show your connected reader

Loose ends:
- I am having a little trouble with the keep searching flow and multiple readers. I'll resolve this during the review

Update release notes:
